core/src/main/java/io/aiven/klaw/service/EmailService.java

```diff
@@ -111,6 +93,42 @@ public void sendSimpleMessage(
               + "\t</tr>\n"
               + "</table>";
 
+      if (notificationHeaderLogo != null) {
+        if (notificationHeaderLogo.equals("http://yourcompany/logo.png")) {
+          notificationHeaderLogo = KW_LOGO_PNG;
+        }
+      } else {
+        notificationHeaderLogo = KW_LOGO_PNG;
+      }
+
+      String headerString =
```

Comment:
I'm just wondering would headerString and footerString be better created as static template constants e.g.

private static headerTemplate = ....

then use String.format to set them with the value in the constructor as the values shouldn't change after that?

Reply:
Updating it on startup. Format does not seem to work on that malformed string.